internal/reflect/into_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package reflect_test
+
+import (
+	"context"
+	"errors"
+	"reflect"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	refl "github.com/hashicorp/terraform-plugin-framework/internal/reflect"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-go/tftypes"
+)
+
+func TestInto_Slices(t *testing.T) {
+	t.Parallel()
+
+	// Just using this to get the reflected type for the odd diagnostics :-)
+	var stringSlice []string
```

Comment:
Pretty sure you can skip this via `reflect.TypeOf([]string{})`

https://go.dev/play/p/tfisl0oH-0s

Less mental redirection for the win!

Reply:
⚡